### PR TITLE
Add Install step in Installation for Linux.

### PR DESCRIPTION
**Pull request by raix852**
- Add Install step in in Installation for Linux for better following installation.
- Fix layout in RNN tutorials.

### CODE REVIEW

File: tensorflow/g3doc/get_started/os_setup.md [hunk 1]
```diff
@@ -623,6 +623,8 @@ Every time you change the Cuda library paths you need to run this step again bef
 you invoke the bazel build command. For the cuDNN libraries, use '6.5' for R2, '7.0'
 for R3, and '4.0.4' for R4-RC.
 
+#### Install
+[Create the pip package and install](#create-the-pip-package-and-install)
 
 ##### Build your target with GPU support
 From the root of your source tree, run:
```
Comment (vrv) [inline tensorflow/g3doc/get_started/os_setup.md]: Actually, I think this next section here is redundant, and would propose removing it.  Thoughts @martinwicke  ?

File: tensorflow/g3doc/get_started/os_setup.md [hunk 2]
```diff
@@ -623,6 +623,8 @@ Every time you change the Cuda library paths you need to run this step again bef
 you invoke the bazel build command. For the cuDNN libraries, use '6.5' for R2, '7.0'
 for R3, and '4.0.4' for R4-RC.
 
+#### Install
+[Create the pip package and install](#create-the-pip-package-and-install)
```
Comment (martinwicke) [inline tensorflow/g3doc/get_started/os_setup.md]: Add a period at the end?

File: tensorflow/g3doc/get_started/os_setup.md (hunk 1 above)
Comment (martinwicke) [inline tensorflow/g3doc/get_started/os_setup.md]: Yes, it's redundant. Also the ./configure step should just be moved to the "Configure the installation" section, to replace the description of the configure step there.

**Comment by raix852** [comment]
Merged configure the installation as suggested. But I didn't have an experience on configure GPU because lacked of environment.  So it maybe have some incorrect parts.

**Comment by martinwicke** [comment]
I think this is fine. Reading the whole thing I noticed it's still pretty garbled, but this is a good start. We should probably rewrite this to factor the GPU story out completely -- it's currently repeated between MacOS and Linux.

Anyway, LGTM.

**Comment by martinwicke** [comment]
Jenkins, test this please.

**Pull request merged**